coordinator/src/main/java/com/tencent/rss/coordinator/SimpleClusterManager.java

```diff
@@ -72,14 +72,18 @@ public SimpleClusterManager(CoordinatorConf conf) {
     }
   }
 
-  private void nodesCheck() {
+  @VisibleForTesting
+  protected void nodesCheck() {
```

Comment:
I prefer `default access modifier` than `protected access modifier`. Test code can access the same package source code.